content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+description: >
+    What are Operators
+---
+
+## What is an Operator?
+
+Operators make it easy to manage complex stateful applications on top of Kubernetes. By [Extending the Kubernetes API](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) with CustomResourceDefinitions it is possible to create what we call Operator Projects where you will be able to create and define your own API's to represent your solutions on the cluster and manage them by looking for the desired state. For a better understanding, see the [Operator Pattern][operator-pattern].
+
+## How are Operators and Controllers related?
+
+See that following the [Operator's pattern][operator-pattern]; you will create [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) which provides a reconcile function responsible for synchronizing the resources until a desired state on the cluster. For a more practical explanation, see also [What’s in a controller?](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller).
+
+## What is Operator Lifecycle Manager?
+
+Operator Lifecycle Manager (OLM) helps users install, update, and manage the lifecycle of all Operators and their associated services running across their clusters. This project is a component of the [Operator Framework][operator-framework], an open-source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the introduction [blog post](https://www.openshift.com/blog/introducing-the-operator-framework?extIdCarryOver=true&sc_cid=701f2000001Css5AAC).
+
+## How does OLM enable cluster admins and developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.x, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform (OCP) web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+
+For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring, and big data services without having to be subject matter experts, because the Operator has that knowledge baked into it.
+
+## Build with the Operator SDK
```

Comment:
From the blog post mentioned this is doc: How do I start writing an Operator? 
seems like a more appropriate question to start off this section?